x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/SqlDataTypes.java

```diff
@@ -582,7 +582,7 @@ public static int displaySize(DataType dataType) {
             return 29;
         }
         if (dataType == IP) {
```

Comment:
`DataType.size()` could be potentially returned since it's the same value.

In fact, maybe this approach could be used for other DataTypes as well if applicable - that is whenever size == displaySize